roles/nginxplus/files/fail2ban/nginx-f_inclusive.conf

```diff
@@ -0,0 +1,4 @@
+nginx-f_inclusive.conf
```

Comment:
We need to remove this first line,  otherwise I get the error:

```
Wrong config file: File contains no section headers.
file: '/Users/sandbergj/repos/princeton_ansible/roles/nginxplus/files/fail2ban/nginx-f_inclusive.conf', line: 1
'nginx-f_inclusive.conf\n'
```